client/src/components/landing/components/LandingTop.js

```diff
@@ -17,6 +20,7 @@ const propTypes = {
   page: PropTypes.string
 };
 
+const { clientLocale } = envData;
 function LandingTop({ page }) {
   const { t } = useTranslation();
 
```

Comment:
```suggestion
  const showChineseLogos = ['chinese', 'chinese-tradition'].includes(
    clientLocale
  );
```
`('chinese' || 'chinese-traditional')` evaluates to `'chinese'` since it's truthy, so the new logos would not appear for the `chinese-traditional`.

This expression is a bit bigger, so let's pull it outside the rendered part.